common/achievement/evaluate.ts

```diff
@@ -27,7 +27,6 @@ async function _evaluateAchievement(
         where: {
             userId,
             metric: { in: metrics },
-            AND: relation ? { relation: { startsWith: relation } } : {},
```

Comment:
Unfortunately, I cannot think of a better way right now, but the impact should not be very limited.